pyproject.toml

```diff
@@ -6,16 +6,17 @@ authors = ["Martin Packman <martin.packman@visual-meaning.com>"]
 description = "Convert tabular data into triples."
 
 [tool.poetry.dependencies]
-python = "^3.11"
-pycountry = "^20"
+python = "^3.8.10"
```

Comment:
`3.11` was your will on the pairing session, hun @bz2

Reply:
Indeed - and updating to latest makes a lot of sense for our application use. The rethink is as this repo is a library - so keeping a wider range of supported versions is reasonable.

Reply:
@bz2, why explicitly `3.8.10` rather than `3.8`?

Reply:
That is also a good question - my first version of this said `^3.8` which poetry complained about in dependency resolution because `rdflib` has _its_ minversion as `3.8.1` so a narrower range. So, I just set our min version to the last security update (from 2021).